packages/realm-web/src/transports/PrefixedTransport.ts

```diff
@@ -0,0 +1,31 @@
+import { Transport, Request } from "./Transport";
+
+/**
+ * Prefixes all request with a path prefix
+ */
+export class PrefixedTransport implements Transport {
+    private readonly fetcher: Transport;
+    private readonly pathPrefix: string;
+
+    constructor(fetcher: Transport, pathPrefix: string) {
```

Comment:
can we rename this to `transport` much better later when we do `this.fetcher.fetch` to be `this.transport.fetch`. `fetcher` name leaks the implementation details.

Reply:
I think this is just a left-over from when "Transport" was named "Fetcher" 👍